tasks/linux.sh

```diff
@@ -161,4 +113,45 @@ case "$available_manager" in
         cmd_status="$(yum_status "\{ \"old_version\": ${old_version}, \"version\": \"%{VERSION}-%{RELEASE}\" \}")"
         success "$cmd_status"
     esac
+    ;;
+
+  "zypper")
+    # Non-interactive and no color
+    options+=("-n")
+
+    # <package>-<version> is the syntax for installing a specific version in zypper
+    [[ $version ]] && name="${name}-${version}"
+
+    case "$action" in
+      "install")
+        zyp_cmd="in"
+        ;;
+
+      # zypper won't upgrade a package if the newer version comes from a different vendor
+      # So we'll use zypper in with the latest version
+      "upgrade")
+        # Call this to get the candidate and installed version
+        zypper_status "${name%%=*}" >/dev/null
+        old_version="$installed_version"
+
+        [[ $candidate ]] && name="${name}-${candidate}"
+        zyp_cmd="in"
+        ;;
+
+      "uninstall")
+        zyp_cmd="rm"
+        ;;
+
+      "status")
+        :
```

Comment:
Could maybe comment about this noop given that status will always be handled below.

Reply:
Now that I think about it, we don't need this element of the case if it doesn't do anything.  I'll just remove it.